modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/util/NotificationTemplateUtil.java

```diff
@@ -79,13 +81,32 @@ public static NotificationContext createNotificationContext(
 		User user, String body, String description, String subject,
 		String type) {
 
-		return createNotificationContext(
-			user, body, description,
-			Collections.singletonList(
+		List<NotificationRecipientSetting> notificationRecipientSettings =
+			new ArrayList<>();
+
+		if (type.equals(NotificationConstants.TYPE_USER_NOTIFICATION)) {
+			notificationRecipientSettings = Collections.singletonList(
+				createNotificationRecipientSetting(
+					"userScreenName", user.getScreenName()));
+		}
+		else if (type.equals(NotificationConstants.TYPE_EMAIL)) {
+			notificationRecipientSettings = Arrays.asList(
```

Comment:
@seltong For email it is necessary that all this information like bcc, cc, from, fromName, single Recipient, to are necessary, because before the check that I created there is a check to validate that these standard fields exist.